File: ProyectoFinal/ProyectoFinal/Parametros/Salida/ListarModulosPorTipoDeUsuarioOut.cs
using ProyectoFinal._2_Dominio.Entidades;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal.Parametros.Salida
{
    public class ListarModulosPorTipoDeUsuarioOut : ParametroBaseOut
    {
        public List<Modulo> Modulos { get; set; }
    }
}

File: Interfaz/ProyectoFinal/src/app/Parametros/Salida/ListarModulosPorTipoDeUsuarioOut.ts
import { Modulo } from "src/app/Components/Modulo/class/modulo";
import { ParametroBaseOut } from "./ParametroBaseOut";
export class ListarModulosPorTipoDeUsuarioOut extends ParametroBaseOut{
    
    private modulos: Modulo[] = [];

    constructor()
    {
        super();
    }

    public get Modulos(): Modulo[] {
        return this.modulos;
    }
    public set Modulos(value: Modulo[]) {
        this.modulos = value;
    }
}